Create second_newsletter.md (#223)
alan-turing-institute/the-turing-way: project_management/second_newsletter.md

# The Turing Way - February update

Dear friends of the Turing Way,

It's been almost a month since we announced the project and we've made great progress since then that we think you should know about.

## Workshops
We still have spots available for our workshops in March. 
We can provide financial support to cover travel, accommodation or childcare costs to make it easier for you to attend. 
Please get in touch at theturingway@gmail.com to let us know how we can support you.

> Note the start and end times, you're welcome to hang out for an extra hour but we'll officially finish at 4pm so that anyone with childcare responsibilities can get home!

Reviewer: ```suggestion
We start at 10am with coffee from 9:30am and finish at 4pm.
You're welcome to hang out for an extra hour until 5pm but we'll officially finish at 4pm so that anyone with childcare responsibilities can get home!
```

* 2.00 - 3.30: Build your own Binder (with coffee from 2.00)
* 3.30 - 4.00: Demonstrating your Binder, general questions, feedback and close
* 4.00 - 5.00: Optional hangout with instructors

**Workshop locations and sign-up**
* [Manchester - 1st March](https://www.eventbrite.co.uk/e/boost-your-research-reproducibility-with-binder-manchester-registration-55331997494)
* [London - 12th March](https://www.eventbrite.co.uk/e/boost-your-research-reproducibility-with-binder-london-registration-55337162944)

### Build a BinderHub workshop

During this free workshop we will demonstrate how to build your own BinderHub on Microsoft Azure cloud computing resources. 
We will help you get started with building a BinderHub on your institution's computing platform and discuss the challenges of maintaining a BinderHub. 
At the end of the workshop you will know why this would be a useful resource for your team, and will know where to look for help and support building your institution's BinderHub.

* 9.30 - 10.00: Registration, coffee and introductions
* 10.00 - 10.30: Introduction to the workshop and The Turing Way
* 10.30 - 11.00: Introduction to Binder and why *you* should build a BinderHub

> * 11.00 - 12.00: Practical session – Build your BinderHub

Reviewer: ```suggestion
* 11.00 - 12.00: Practical session – Build a BinderHub
```

* 12.00 - 1.00: Lunch

> * 1.00 - 2.45: Practical session – Build your BinderHub

Reviewer: ```suggestion
* 1.00 - 2.45: Practical session – Build a BinderHub
```

* 2.45 - 3.00: Coffee
* 3.00 - 4.00: Looking forwards: Progress review, lessons learned and future strategies
* 4.00 - 5.00: Optional hangout with instructors

**Workshop location and sign-up**
* [Sheffield - 18th March](https://www.eventbrite.co.uk/e/build-a-binderhub-registration-55336756729)

## Handbook

Three chapters are written and ready for your comments! 

> Version control, reproducibility and Open Research are ready for feedback. 

Reviewer: Add links to chapters?

Does "Open Research" have to be capitalised? I think either make all chapters capitalised, or none would look better.

Author: just so used to capitalising Open Research, now captalised all of the chapters

We're currently working on the chapters on research data management, reproducible computing environments, testing and continuous integration.

> You can follow the progress through the [pull requests on our GitHub repsoitory](https://github.com/alan-turing-institute/the-turing-way/pulls).

Reviewer: ```suggestion
You can follow the progress through the [pull requests on our GitHub repsoitory](https://github.com/alan-turing-institute/the-turing-way/pulls): [#146](https://github.com/alan-turing-institute/the-turing-way/pull/146), [#147](https://github.com/alan-turing-institute/the-turing-way/pull/147) & [#148](https://github.com/alan-turing-institute/the-turing-way/pull/148).
```

> We will review our contributing guidelines to make sure it's easy for you to feed back and add your knowledge to the Turing Way. 

Reviewer: ```suggestion
We're still building up our [contributing guidelines](https://github.com/alan-turing-institute/the-turing-way) to make sure it's easy for you to feed back and add your knowledge to the Turing Way. 
```

In the meantime, creating an issue on [GitHub](https://github.com/alan-turing-institute/the-turing-way/issues) or pinging us on [our gitter channel](https://gitter.im/alan-turing-institute/the-turing-way) is always a good start!

## Talks
The team has been spreading the word about the Turing Way project through various talks:
* Kirstie Whitaker presented at the Cambridge Psychiatry PI Day on 30 January 2019. 

> Her	slides can be found on [figshare](https://doi.org/10.6084/m9.figshare.7649156.v2).

Reviewer: ```suggestion
Her slides can be found on [figshare](https://doi.org/10.6084/m9.figshare.7649156).
```

### Build a BinderHub workshop

During this free workshop we will demonstrate how to build your own BinderHub on Microsoft Azure cloud computing resources. 
We will help you get started with building a BinderHub on your institution's computing platform and discuss the challenges of maintaining a BinderHub. 
At the end of the workshop you will know why this would be a useful resource for your team, and will know where to look for help and support building your institution's BinderHub.

* 9.30 - 10.00: Registration, coffee and introductions
* 10.00 - 10.30: Introduction to the workshop and The Turing Way
* 10.30 - 11.00: Introduction to Binder and why *you* should build a BinderHub
* 11.00 - 12.00: Practical session – Build a BinderHub
* 12.00 - 1.00: Lunch
* 1.00 - 2.45: Practical session – Build a BinderHub
* 2.45 - 3.00: Coffee
* 3.00 - 4.00: Looking forwards: Progress review, lessons learned and future strategies
* 4.00 - 5.00: Optional hangout with instructors

**Workshop location and sign-up**
* [Sheffield - 18th March](https://www.eventbrite.co.uk/e/build-a-binderhub-registration-55336756729)

## Handbook

Three chapters are written and ready for your comments! 

> Version Control, Reproducibility and Open Research are ready for feedback. 

Reviewer: Can we add a link to them here? Each individually, or just to the chapters folder?

> We're currently working on the chapters on Research Data Management, Reproducible Computing Environments, Testing and Continuous Integration.

Reviewer 2: Are Testing and Continuous Integration separate chapters? I'm wondering is this line should read "We're currently working on the chapters on Research Data Management, Reproducible Computing Environments, and Testing and Continuous Integration."

Reviewer: They are separate chapters, so maybe:

```suggestion
We're currently working on the chapters on Research Data Management, Reproducible Computing Environments, Testing, and Continuous Integration.
```

Author: They are as far as I understood (pretty sure there are 3 WIP PRs), but an Oxford comma will clarify things!

> You can follow the progress through the [pull requests on our GitHub repsoitory](https://github.com/alan-turing-institute/the-turing-way/pulls): [#146](https://github.com/alan-turing-institute/the-turing-way/pull/146), [#147](https://github.com/alan-turing-institute/the-turing-way/pull/147) & [#148](https://github.com/alan-turing-institute/the-turing-way/pull/148).

Reviewer: I missed the chapter for RDM:
```suggestion
You can follow the progress through the [pull requests on our GitHub repsoitory](https://github.com/alan-turing-institute/the-turing-way/pulls): [#196](https://github.com/alan-turing-institute/the-turing-way/pull/196), [#146](https://github.com/alan-turing-institute/the-turing-way/pull/146), [#147](https://github.com/alan-turing-institute/the-turing-way/pull/147) & [#148](https://github.com/alan-turing-institute/the-turing-way/pull/148).
```

In the meantime, creating an issue on [GitHub](https://github.com/alan-turing-institute/the-turing-way/issues) or pinging us on [our gitter channel](https://gitter.im/alan-turing-institute/the-turing-way) is always a good start!

## Talks
The team has been spreading the word about the Turing Way project through various talks:
* Kirstie Whitaker presented at the Cambridge Psychiatry PI Day on 30 January 2019. 
Her slides can be found on [figshare](https://doi.org/10.6084/m9.figshare.7649156).
* Sarah Gibson presented at the [UKRI Cloud workshop](https://cloud.ac.uk/workshops/feb2019/). 
Resources for her talk can be found [here](https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/presentations/UKRI_cloud_2019-02-12/UKRI_cloud_demo_2019-02-12.md).
* Patricia Herterich introduced the Turing Way to the Birmingham Hacker Within community on 18 February 2019. 
Slides for her talk are published on [zenodo](http://doi.org/10.5281/zenodo.2566430).

## Where to meet us in the next weeks

If you can't make it to one of our workshops - we will attend a few other other events around the UK where you can chat to us in person.

> * Kirstie will be at the [Turing Lecture: Open source influences on technology innovation](https://www.turing.ac.uk/events/turing-lecture-open-source-influences-technology-innovation) at The Royal Society of Medicine, London on 27 February 2019

Reviewer: ```suggestion
* Kirstie will be at the [Turing Lecture: Open source influences on technology innovation](https://www.turing.ac.uk/events/turing-lecture-open-source-influences-technology-innovation) at The Royal Society of Medicine, London on 27 February 2019.
  There are still tickets available if you'd like to come!
```
